src/devices/cpu/evolution/evo.h

```diff
@@ -0,0 +1,50 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+
+#ifndef MAME_CPU_EVO_EVO_H
+#define MAME_CPU_EVO_EVO_H
+
+#pragma once
+
+enum
+{
+	EVO_PC = STATE_GENPC
+};
+
+class evo_cpu_device : public cpu_device
+{
+public:
+	evo_cpu_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+protected:
+	virtual void device_start() override;
+	virtual void device_reset() override;
+
+	virtual uint32_t execute_min_cycles() const noexcept override { return 5; }
+	virtual uint32_t execute_max_cycles() const noexcept override { return 5; }
```

Comment:
This is potentially going to do strange things when the loop in `execute_run` has granularity of 1 while you’re returning 5 from `execute_min_cycles`.